examples/langextract_example.py

```
@@ -0,0 +1,275 @@
+    
+    # Extract and visualize
+    result = langextract_extract(
+        text=contract_text,
+        extractions=key_terms,
+        document_id="consulting-agreement",
+        output_path="contract_analysis.html",
+        auto_open=False  # Set to True to open in browser automatically
+    )
+    
+    if result.get("success"):
+        print(f"✅ Analysis complete!")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)